go/flags/endtoend/vtctld.txt

```diff
@@ -145,7 +130,6 @@ Usage of vtctld:
       --topo_k8s_context string                                          The kubeconfig context to use, overrides the 'current-context' from the config
       --topo_k8s_kubeconfig string                                       Path to a valid kubeconfig file. When running as a k8s pod inside the same cluster you wish to use as the topo, you may omit this and the below arguments, and Vitess is capable of auto-discovering the correct values. https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
       --topo_k8s_namespace string                                        The kubernetes namespace to use for all objects. Default comes from the context or in-cluster config
-      --topo_read_concurrency int                                        Concurrency of topo reads. (default 32)
```

Comment:
why did this disappear? I can't see the corresponding change to where we register this flag.

Reply:
ok I see it. It's in healthcheck.go. We'll need to bring that back until we actually delete the vtctld web UI in v16.